tycho-client/README.md

```diff
@@ -325,9 +325,9 @@ If you wish to use tycho as a service instead of hosting it yourself, the follow
 
 ### PropellerHeads (Beta)
 
-|                        |                                                         |
-|------------------------|---------------------------------------------------------|
-| **URL**                | *tycho-beta.propellerheads.xyz*                         |
-| **RPC Docs**           | *[View Documentation](https://tycho-beta.propellerheads.xyz/docs/)* |
+|                        |                                                                                                                                                  |
```

Comment:
I have no idea what happened here -.- 
But the readme seems fine

Reply:
Can you undo this formatting please?